unified-codes/libs/data/src/lib/types/Entity.ts

```diff
@@ -21,20 +28,23 @@ export interface IEntity {
   interactions?: IDrugInteraction[];
   children?: IEntity[];
   properties?: IProperty[];
+  parent?: IEntityParent[];
```

Comment:
I kind of am also confused/uncertain about this new type - technically, in this case, the shape of this object would be the same as `IEntity`? Or is there something special about this one? 🤔